lib/modules/manager/tekton/extract.ts

```diff
@@ -86,7 +86,9 @@ function getDeps(doc: TektonResource): PackageDependency[] {
   return deps;
 }
 
-const taskAnnotation = regEx(/^pipelinesascode\.tekton\.dev\/task(-[0-9]+)?$/);
+const annotationRegex = regEx(
+  /^pipelinesascode\.tekton\.dev\/(task|pipeline)(-[0-9]+)?$/,
```

Comment:
```suggestion
  /^pipelinesascode\.tekton\.dev\/(?:task(-[0-9]+)?|pipeline)$/,
```
this is safer, I've read the docs. 🤗

Reply:
Wow, yea, they changed it 5 days after I made this PR